main.py

```diff
@@ -0,0 +1,11 @@
+from flask import Flask
+
+app = Flask(__name__)
```

Comment:
Let's put a separate commit in the README.md on how to run the Flask server @elainoz @kvg-png  @ntresvalles1  @naylynn-TR  this is where the sql alchemy code will come; 

You will have to instantiate the sql flask db here 

`app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:////tmp/test.db'
db = SQLAlchemy(app)`